### PR TITLE
[Impeller] Skip TextRotated golden test.

### PR DESCRIPTION
**Pull request by dnfield**
This test ends up with small differences depending on which mac host it runs on and will end up blocking rollers.

### CODE REVIEW

**Review by chinmaygarde — APPROVED** [review]
The insight from the Skia team was that the golden tests were to be verified out of band by the triagers and not in the critical path for rollers. Perhaps we should follow a similar approach. The test is unlikely to ever offer pixel perfect reproducibility but disabling it gives us nothing. Disabling the test entirely to appease the roller is perhaps throwing the baby out with the bathwater.

LGTM if this is to get the rollers going but we should consider holding doing this and instead stop the rollers from tripping on goldens.

**Comment by dnfield** [comment]
(and yes, the goal of this PR is to make sure rollers don't get stuck on golden approvals)

**Comment by gaaclarke** [comment]
@chinmaygarde @dnfield there is parameter when we submit to skia gold about how strict we want the golden test to be.  We could potentially lower the threshold.

**Comment by Piinks** [comment]
> We could potentially lower the threshold.

I think on some platforms this is already done (web I think?) via the fuzzy matching algorithm. So we could add it for the mac platform probably.

@mdebbar knows better the config for the engine Gold instance

**Comment by chinmaygarde** [comment]
I'd rather just follow the Skia GPU teams recommendations on their use of Gold and have this be out of band.

**Pull request merged**
